weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -187,33 +191,48 @@ const useCall = (
 
   const [callRes, setCallRes] =
     useState<traceServerTypes.TraceCallReadRes | null>(null);
-  const doFetch = useCallback(() => {
-    if (deepKey) {
-      loadingRef.current = true;
-      setCallRes(null);
-      getTsClient()
-        .callRead({
-          project_id: projectIdFromParts(deepKey),
-          id: deepKey.callId,
-          include_costs: opts?.includeCosts,
-          ...(opts?.includeTotalStorageSize
-            ? {include_total_storage_size: true}
-            : null),
-        })
-        .then(res => {
-          loadingRef.current = false;
-          setCallRes(res);
-        });
-    }
-  }, [deepKey, getTsClient, opts?.includeCosts, opts?.includeTotalStorageSize]);
+  const doFetch = useCallback(
+    (invalidateCache = false) => {
```

Comment:
I think this is cleaner with `invalidateCache` as a named argument, eg
```suggestion
    ({invalidateCache = false}: {invalidateCache?: boolean} = {}) => {
```

and then called as `doFetch({invalidateCache: true})` or `doFetch({invalidateCache: false})`